src/Native/System.Native/pal_process.h

```diff
@@ -5,6 +5,9 @@
 
 #include "pal_types.h"
 #include <string.h>
+#include <sys/param.h>
+
+struct rusage_info_v3;
```

Comment:
The underlying function takes in a ```rusage_info_t``` which is typedef'd to ```void*``` but is actually an ```rusage_info_v3``` under the covers; there is also v1 and v2 versions but those are deprecated.